parse/table_definitions_arbitrum/uniswap/UniswapV3Factory_event_PoolCreated.json

```diff
@@ -0,0 +1,76 @@
+{
```

Comment:
You also need to add `UniswapV3Pool_event_Swap`, like for Optimism here: https://github.com/nansen-ai/evmchain-etl-table-definitions/tree/main/parse/table_definitions_optimism/uniswap

I don't think we need the events for `OwnerChanged` and `FeeAmounEnabled` for our use case, but maybe nice to have?

Reply:
Yep, but the swap should be ideally done in a separate PR since it references UniswapV3Factory_event_PoolCreated that isn't a table yet

Reply:
Tables are build, can we now create a PR for UniswapV3Pool_event_Swap cc: @kris947

Reply:
Aha, I have actually never done this myself so I'm learning too 😉